### PR TITLE
Minikube script updates

### PR DESCRIPTION
**Pull request by rnc**

### CODE REVIEW

File: deploy/minikube-development.sh [hunk 1]
```diff
@@ -2,37 +2,42 @@
...
+kubectl apply -k $DIR/crds/base
+#Load missing CRD
+kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.69.1/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml
+sleep 2
 
 $DIR/base-development.sh  $1
 
 # base-development.sh switches to the test-jvm-namespace namespace
-kubectl create sa pipeline
+kubectl create --dry-run=client -o=yaml sa pipeline | kubectl apply -f -
```
Comment (stuartwdouglas) [inline deploy/minikube-development.sh:34]: Why this change?

Comment (rnc) [reply deploy/minikube-development.sh:34]: This is similar to prior changes I made in other scripts - this way, if the script is run twice, it is not an error condition and instead simply configuring if there are any changes, ignoring if there are not. (create versus apply).

Comment (stuartwdouglas) [reply deploy/minikube-development.sh:34]: ah, I missed that had changed.
